### DISCUSSION TITLE
Mission salvageartifactabyss will select artifact inside your submarine as mission target

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15424] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

the mission selects your own artifact as mission target

### Reproduction steps

1. have artifact inside your own submarine
2. pick mission salvageartifactabyss
3. infrasonic signal locates your own artifact

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
